packages/core/src/codewhisperer/service/transformByQ/transformMavenHandler.ts

```diff
@@ -108,6 +108,8 @@ function copyProjectDependencies(dependenciesFolder: FolderInfo, modulePath: str
 }
 
 export async function prepareProjectDependencies(dependenciesFolder: FolderInfo, rootPomPath: string) {
+    // use await to allow other chat messages to send as well, before the build starts
+    await vscode.window.showInformationMessage(CodeWhispererConstants.buildStartedNotification)
```

Comment:
We used to have this here actually, months ago, but somehow it must have been removed probably in favor of the chat. For most things, right now `/transform` shows updates both as notifications and as chat messages, so this is fine, and we should be sending this notification here anyway.

It seems that `copyProjectDependencies` and `installProjectDependencies` prevent the numerous chat messages from displaying before they begin, so we need some kind of delay just before they execute to allow all of these chat messages to be sent before the copy / install functions start. Something like `await sleep(500)` will sleep for 500ms for all users like you mentioned, but adding this notification here seems to also solve the problem. Not yet sure if this is a perfect solution, but again, we should be showing this notification here anyway.